src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -357,7 +373,7 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
           labelInline={labelInline}
           labelWidth={labelWidth}
           labelAlign={labelAlign}
-          labelHelp={labelHelp}
+          labelHelp={!validationRedesignOptIn && labelHelp}
```

Comment:
comment: technically this is a breaking change as in theory someone _could_ have used this prop with the new validation pattern. I'll have a little investigation on this tomorrow and get back to you on it

Reply:
I added this due to the tooltip and the hint text both being rendered with the same `labelHelp` prop, this is just meant to stop the tooltip being rendered along with the hint text as they would have the same text value.

Reply:
Yeah makes sense, technically it's a breaking change but tbh no one should have used it when using the new pattern and their project won't break now with the guard added, it just won't render the help etc, so let's leave it as is